tools/vms-generator/utils/utils.go

```diff
@@ -122,6 +123,27 @@ func getBaseVMI(name string) *v1.VirtualMachineInstance {
 func initFedora(spec *v1.VirtualMachineInstanceSpec) *v1.VirtualMachineInstanceSpec {
 	addContainerDisk(spec, fmt.Sprintf("%s/%s:%s", DockerPrefix, imageFedora, DockerTag), busVirtio)
 	addRNG(spec) // without RNG, newer fedora images may hang waiting for entropy sources
+
+	return spec
+}
+
+func addINterface(spec *v1.VirtualMachineInstanceSpec, bindingMethod v1.InterfaceBindingMethod) *v1.VirtualMachineInstanceSpec {
```

Comment:
why both I and N upper case in the name of the function?

Reply:
Also, the name of the function doesn't fully reflect its intent: it is named like a function that allows to add any interface, but in reality it's only fitting for "default" interfaces. Please rename accordingly, e.g.: `addDefaultInterface`.

Reply:
Also, the function doesn't *add* but *override* the list of interfaces. Please either rename into: `setDefaultInterface` or change the body to `append` to an existing slice (if it is not `nil` of course).